2023-03-21 22:16:31,383 INFO    MainThread:23300 [wandb_setup.py:_flush():68] Configure stats pid to 23300
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_setup.py:_flush():68] Loading settings from C:\Users\chris\.config\wandb\settings
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_setup.py:_flush():68] Loading settings from c:\Users\chris\OneDrive\Skrivebord\huggingFace\wandb\settings
2023-03-21 22:16:31,384 WARNING MainThread:23300 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_WATCH
2023-03-21 22:16:31,384 WARNING MainThread:23300 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_LOG_MODEL
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'project': 'Sailboat FGVC'}
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_init.py:_log_setup():476] Logging user logs to c:\Users\chris\OneDrive\Skrivebord\huggingFace\wandb\run-20230321_221631-3or861hp\logs\debug.log
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_init.py:_log_setup():477] Logging internal logs to c:\Users\chris\OneDrive\Skrivebord\huggingFace\wandb\run-20230321_221631-3or861hp\logs\debug-internal.log
2023-03-21 22:16:31,384 INFO    MainThread:23300 [wandb_init.py:init():516] calling init triggers
2023-03-21 22:16:31,385 INFO    MainThread:23300 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-03-21 22:16:31,385 INFO    MainThread:23300 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 1mdwiph9
2023-03-21 22:16:31,386 INFO    MainThread:23300 [wandb_run.py:_finish():1746] finishing run cringgaard/Sailboat FGVC/1mdwiph9
2023-03-21 22:16:31,388 ERROR   MainThread:23300 [jupyter.py:save_history():440] Run pip install nbformat to save notebook history
2023-03-21 22:16:31,388 INFO    MainThread:23300 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-03-21 22:16:31,389 INFO    MainThread:23300 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-03-21 22:16:31,389 INFO    MainThread:23300 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-03-21 22:16:31,389 INFO    MainThread:23300 [wandb_run.py:_restore():1932] restore
2023-03-21 22:16:31,389 INFO    MainThread:23300 [wandb_run.py:_restore():1938] restore done
